tests/components/ViewedByCustomer/ViewedByCustomer.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { ViewedByCustomer } from '../../../src/Index';
+import { Mock, Fake } from 'coveo-search-ui-tests';
+import { IViewedByCustomerOptions } from '../../../src/components/ViewedByCustomer/ViewedByCustomer';
+import { IQueryResult } from 'coveo-search-ui';
+import { AdvancedComponentSetupOptions } from 'coveo-search-ui-tests/MockEnvironment';
+
+describe('ViewedByCustomer', () => {
+    const VIEWED_BY_CUSTOMER_LABEL_CSS_CLASS = 'viewed-by-customer-label';
```

Comment:
Is it necessary to prefix with VIEWED_BY_CUSTOMER? I feel like, since we are in the context of a test file for ViewedByCustomer, it makes the asserts too verbose